_source/atoms/headline/HeadlineContainer.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import Component from './Headline';
+
+export const mapStateToProps = (state) => ({
+  darkMode: state.user.darkMode
+});
+
+const Container = connect(
+  mapStateToProps
+)(Component);
```

Comment:
Could you explain why we need to do it in two steps here? As Container just has mapStateProps, why do we need it at all?